internal/bminventory/inventory.go

```diff
@@ -6751,3 +6782,16 @@ func getDefaultHighAvailabilityAndMasterCountParams(highAvailabilityMode *string
 	// both are set
 	return highAvailabilityMode, controlPlaneCount
 }
+
+func extractMirroredRegistriesFromConfig(log logrus.FieldLogger, mirrorRegistryConfig *common.MirrorRegistryConfiguration) []string {
+	var ignorableRegistries []string
+	mirroredRegistries, err := mirrorregistries.ExtractLocationMirrorDataFromRegistriesFromToml(mirrorRegistryConfig.RegistriesConf)
+	if err != nil {
+		log.WithError(err).Warnf("failed to extract location from saved cluster mirror registry configuration while updating cluster params")
+	}
+	for _, registry := range mirroredRegistries {
+		log.Infof("CRYSTAL adding registry to ignorable %s", registry)
```

Comment:
This looks like a debug message that needs to be removed.

Reply:
Good catch, thanks! Removed it